aspnetcore/security/data-protection/introduction.md

```diff
@@ -18,9 +18,17 @@ Web apps often need to store sensitive data. The Windows data protection API ([D
 
 The ASP.NET Core data protection stack was designed to:
 
-* Provide a built in solution for most Web scenarios.
-* Address many of the deficiencies of the previous encryption system. 
-* Serve as the replacement for the `<machineKey>` element in ASP.NET 1.x - 4.x.
+* Provide a built in solution for most Web scenarios, including web farms and Azure Container Apps.
+* Address many of the deficiencies of the previous encryption system.
+
+## Key consistency in web farms and Azure Container Apps
+
+If the developer has not configured data protection in ASP.NET Core, the environment variable `autoConfigureDataProtection` is set to `true`.  When `autoConfigureDataProtection` is set to `true`, the data protection keys are read only. Setting the data [configuration](xref:fundamentals/configuration/index) keys to read only is designed to maintain data consistency in web farms and [Azure Container Apps](/azure/container-apps/overview).
```

Comment:
Personally, I think we should remove this paragraph because I think `autoConfigureDataProtection` is effectively an implementation detail.  The only reason to document it at all is to give experts (and CSS) a (non-trivial) way to change the default behavior.

Having said that, if we're going to keep it, I think some of the technical details need refinement.

Reply:
As an aside: we haven't documented the opt-out process because, at last check (please correct me @claudiaregio), azd hasn't decided how they're going to expose that yet.

Reply:
> Personally, I think we should remove this paragraph because I think autoConfigureDataProtection is effectively an implementation detail. 

Removed. @amcasey  I think I've addressed all your feedback except for 

>As an aside: we haven't documented the opt-out process because, at last check (please correct me @claudiaregio), azd hasn't decided how they're going to expose that yet.